### ISSUE TITLE
README.md link to "Data Hub Framework website" in "Advanced Hub Usage" is broken

### ISSUE THREAD

**Issue by mikrovvelle** [marklogic/marklogic-data-hub#837]
Minor bug in README:

https://github.com/marklogic-community/marklogic-data-hub#advanced-hub-usage

It reads:

```markdown
## Advanced Hub Usage
Ready to go beyond QuickStart and start using DHF in your own Java or Gradle project?
The [Data Hub Framework website](https://marklogic-community.github.io/) also provides that.
```
The link is 404. Looks like it should be:
```markdown
## Advanced Hub Usage
Ready to go beyond QuickStart and start using DHF in your own Java or Gradle project?
The [Data Hub Framework website](https://marklogic-community.github.io/marklogic-data-hub/) also provides that.
```

┆Status: Done

**Comment by dmcassel** [comment]
➤ Mike Wooldridge commented:

This appears to be fixed. The above link goes here:

https://marklogic-community.github.io/marklogic-data-hub/

and does not result in a 404.

**Comment by dmcassel** [comment]
➤ Aries Yuwono commented:

Verified the link that it's fixed

**Comment by popzip** [comment]
Oh, looks like i fixed it [here](https://github.com/marklogic-community/marklogic-data-hub/commit/a29afcbd6ff6bff019121e9d07c04be9921e25a2) as a result of this other [bug](https://github.com/marklogic-community/marklogic-data-hub/issues/864).

**Comment by aebadirad** [comment]
Fixed with release of 3.0.0
